lib/accounts_session/session.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module AccountsSession
+  class Session
+    attr_reader :session_id, :first_name, :last_name, :email, :uuid, :avatar_url
+
+    def initialize(attributes = {})
+      attributes.transform_keys!(&:to_s)
+      @session_id = attributes.fetch("session_id")
+      @first_name = attributes.fetch("first_name")
+      @last_name = attributes.fetch("last_name")
+      @email = attributes.fetch("email")
+      @uuid = attributes.fetch("uuid")
+      @avatar_url = attributes.fetch("avatar_url")
+    end
```

Comment:
Could it be worth subclassing OpenStruct here to make it cleaner? We'd have to check that all these keys are present to get the same exception behavior, but it might be more explicit that way.

Reply:
Open struct is "too" open for me. I like when the interface of the class is clearly defined. In openstruct, you can make a typo in attribute name, and you will not find it out until you call some method using the attribute. But I used Struct with keyword arguments.